pkg/eval/mods/path/path.go

```diff
@@ -176,3 +181,100 @@ func isRegular(path string) bool {
 	fi, err := os.Lstat(path)
 	return err == nil && fi.Mode().IsRegular()
 }
+
+//elvdoc:fn temp-dir
+//
+// ```elvish
+// temp-dir [&dir=$dir] [$pattern]
+// ```
+//
+// Create a temporary, unique, directory and output its name. The default for the `&dir` option is a
```

Comment:
> Create a temporary, unique, directory

The last comma is bogus

Reply:
Not only that, the first comma should be replaced by "and": `Create a temporary and unique directory...`. In fact, I'll go further and state that the "temporary" qualifier should be omitted since the directory is not temporary if the program doesn't explicitly delete it. Which is noted by the last sentence of the documentation.